ttnn/cpp/ttnn/operations/data_movement/clone/device/clone_program_factory.cpp

```diff
@@ -11,141 +11,170 @@ CloneOperation::ProgramFactory::cached_program_t CloneOperation::ProgramFactory:
     const operation_attributes_t& operation_attributes,
     const tensor_args_t& tensor_args,
     tensor_return_value_t& output) {
-    using namespace tt;
-    using namespace tt::tt_metal;
+    using namespace tt::constants;
     using namespace tt::tt_metal::detail;
+    using namespace tt::tt_metal;
+    using namespace tt;
```

Comment:
I thought that auto format was applied to the code in the PR, so it's strange that the order is changing.
If reapplying the format doesn't cause any issues, then it's fine not to make any changes.

Reply:
My answer is: it's not automatically formatted; I just sorted it myself using VS Code's 'Sort Lines' feature (apology for sort of OCD here).

As it not affect much, can I just leave it as it is?